airbyte-integrations/connectors/source-mailchimp/source_mailchimp/client.py

```diff
@@ -63,19 +63,25 @@ def lists(self, state: DefaultDict[str, any] = None) -> Generator[AirbyteMessage
         stream_name = self._LISTS
         date_created = self._get_cursor_or_none(state, stream_name, cursor_field)
 
-        default_params = {"since_date_created": date_created, "sort_field": cursor_field, "sort_dir": "ASC"}
+        default_params = {"sort_field": cursor_field, "sort_dir": "ASC"}
+        if date_created:
+            default_params["since_date_created"] = date_created
+            max_date_created = parser.isoparse(date_created)
+        else:
+            max_date_created = None
+
         offset = 0
-        max_date_created = date_created
         done = False
         while not done:
-            lists_response = self._client.lists.all(dict(default_params, count=self.PAGINATION, offset=offset))["lists"]
+            params = dict(default_params, count=self.PAGINATION, offset=offset)
+            lists_response = self._client.lists.all(**params)["lists"]
```

Comment:
when passing a dictionary and you want it to be parsed as keyword args, it must be preceded by `**`